tests/Unit/Libs/Soa/v1/YousignTest.php

```diff
@@ -51,9 +61,13 @@ public function it_can_be_instantiated(): void
     /**
      * @test
      * @covers      ::initiateSignature
+     * @dataProvider nullPropertiesProvider
+     *
+     * @param string $nullProperty
+     *
      * @return void
      */
-    public function it_initiates_a_procedure(): void
+    public function it_initiates_a_procedure(string $nullProperty): void
```

Comment:
Aggiungerei nel nome `it_initiates_procedure_with_null_properties` oppure nel provider aggiungerei un caso dove tutto è velorizzato, almeno copri tutto

Reply:
No in realtà questo è l'unico test afferente l'inizializzazione di una procedura, ma mi serviva assicurarmi che i dati che vengono passati all'api, sia no "filtrati" per le proprietà null. Però il filtro è testato nel modello, qui mi serve solo assicurarmi che di fatto lo stia usando.